cli/init.go

```diff
@@ -12,11 +11,9 @@ import (
 func init() {
 	// preload manifest so that we have the correct code CID inventory for cli since that doesn't
 	// go through CI
-	if len(build.BuiltinActorsV8Bundle()) > 0 {
-		bs := blockstore.NewMemory()
+	bs := blockstore.NewMemory()
```

Comment:
Is it truly OK to load the bundle into a memory blockstore?

Reply:
here yes, we only need the code CID mapping for the cli process.

Reply:
Gotcha. Seems a bit wasteful, though. Can't we just load the root CID and decode the manifest? (Could do in a follow-up PR too)

Reply:
the blockstore in thrown away, so it is just transient memory usage; below meh threshold ;)

Reply:
Loading 30MiB into a blockstore is going to add unnecessary latency to _every_, _single_, _CLI_, _command. We don't want that.

Reply:
ok, fine, but how fix that? I don't see any other way.

Reply:
plus we are talking a few milliseconds (maybe a whopping 50 of them), is it really such a big deal?

Reply:
adding a TODO for now.

Reply:
I don't see any quick improvements that can be made to avoid this. TODO and move on SGTM.